web/components/TeamList.js

```diff
@@ -1,157 +1,44 @@
-                            fontSize='2.5rem' 
-                        >
-                            Oscar Lafarga
-                        </Text>
-                        <Image 
-                            mr='1rem'
-                            mt='0.38rem'
-                            height='20px' 
-                            src={require('../images/SettingsIcon.png')} 
-                        />
-                    </Flex>
-                </Card>
-                <Card 
-                    bg='purple'
-                    borderRadius={5}
-                    width='250px'
-                    height="30px"
-                    mx='auto'
-                    my='1rem'
-                    depth='-1px 0px 15px rgba(0,0,0,0.65)'
-                >
-                    <Flex 
-                        direction='row'
-                        justifyContent='space-between'
-                        mb='2rem' 
-                        width='250px'
-                        height='50px'
-                        onClick={this.goToPage}
-                    >
-                        <Text
-                            color='black'
-                            ml='0.75rem'
-                            mt='0.25rem'
-                            fontSize='2.5rem' 
-                        >
-                            Quinn Pruitt
-                        </Text>
-                        <Image 
-                            mr='1rem'
-                            mt='0.38rem'
-                            height='20px' 
-                            src={require('../images/SettingsIcon.png')} 
-                        />
-                    </Flex>
-                </Card>
-                <Card 
-                    bg='green'
-                    borderRadius={5}
-                    width='250px'
-                    height="30px"
-                    mx='auto'
-                    my='1rem'
-                    depth='-1px 0px 15px rgba(0,0,0,0.65)'
-                >
-                    <Flex 
-                        direction='row'
-                        justifyContent='space-between'
-                        mb='2rem' 
-                        width='250px'
-                        height='50px'
-                        onClick={this.goToPage}
-                    >
-                        <Text 
-                            color='black'
-                            ml='0.75rem'
-                            mt='0.25rem'
-                            fontSize='2.5rem' 
-                        >
-                            Viki Lafarga
-                        </Text>
-                        <Image 
-                            mr='1rem'
-                            mt='0.38rem'
-                            height='20px' 
-                            src={require('../images/SettingsIcon.png')} 
-                        />
-                    </Flex>
-                </Card>
+                <TeamMember
```

Comment:
Good refactoring into a component to reduce lines of code, but we will also want to use `.map` instead of hard-coded indexed array elements. This will be handled in #8 so no need to make any changes here for now. 👍